client/datasync-client/example/index.ts

```diff
@@ -22,18 +23,17 @@ const task: any = {
 
 $fh.sync.manage(datasetId, options, queryParams, metaData, function() {
   $fh.sync.doCreate(datasetId, task, function(data) {
-    // tslint:disable-next-line:no-console
-    console.log('Data Saved', data);
+    logger.info('Data Saved', data, { tag: 'client:datasync-client:example'});
     $fh.sync.doUpdate(datasetId, data.localId, function(result: any) {
-       // tslint:disable-next-line:no-console
-      console.log('Data updated', result);
+      logger.info('Data updated', result,
+        { tag: 'client:datasync-client:example'});
     }, function(err) {
-       // tslint:disable-next-line:no-console
-      console.log('Error when Saving Data', err);
+      logger.error('Error when Updateing Data', err,
```

Comment:
Ah I meant only change this one to update as this error is related to `doUpdate`, the one below is related to `doCreate`.